src/txn/txn_recover.c

```diff
@@ -449,6 +453,9 @@ __recovery_setup_file(WT_RECOVERY *r, const char *uri, const char *config)
       (WT_IS_MAX_LSN(&r->max_ckpt_lsn) || __wt_log_cmp(&lsn, &r->max_ckpt_lsn) > 0))
         r->max_ckpt_lsn = lsn;
 
+    /* Update the base write gen to the maximum write gen we have seen. */
+    WT_RET(__wt_metadata_update_base_write_gen(r->session, config));
```

Comment:
I suggest for this comment you leave out the implementation details of the function you're calling because if it changes, no one will remember to change this comment. I suggest:
/* Update the base write gen based on this file's configuration. */

Reply:
Changed as suggested